docs/tutorials/trajectories.rst

```diff
@@ -0,0 +1,72 @@
+==========================
+Handle Expert Trajectories
+==========================
+
+For imitation learning we need trajectories.
+Some are generated by experts and serve as demonstrations,
+others are generated by the agent and serve as training data for a discriminator.
+In this library they are stored in a :class:`Trajectory <imitation.data.types.Trajectory>` dataclass:
+
+.. code-block:: python
+
+    @dataclasses.dataclass(frozen=True)
+    class Trajectory:
+        obs: np.ndarray
+        """Observations, shape (trajectory_len + 1, ) + observation_shape."""
+
+        acts: np.ndarray
+        """Actions, shape (trajectory_len, ) + action_shape."""
+
+        infos: Optional[np.ndarray]
+            """An array of info dicts, shape (trajectory_len, )."""
```

Comment:
Either link to what info dicts are for, or briefly explain. This is confusing to someone who doesn't fully understand gym already

Reply:
Good point! I linked to the reference documentation of Trajectory and to the gymnasium documentation.